trulens_eval/trulens_eval/tru.py

```diff
@@ -389,6 +390,47 @@ def Virtual(
 
         return TruVirtual(tru=self, app=app, **kwargs)
 
+    def BenchmarkExperiment(
```

Comment:
why is this camelcased?

Reply:
mainly b/c I view this as a special instance of TruCustomApp - I'm okay for this being snake case if you think it's more appropriate / pythonic

Reply:
Yeah I think the way it is now it makes it seem like a class (I thought it was when I was reading the notebook).

Reply:
@sfc-gh-dkurokawa ah that's b/c it was a class and I forgot I made the change - renamed now